src/base1/Makefile.am

```diff
@@ -55,29 +52,18 @@ dist/base1/mustache.min.js.map: dist/base1/mustache.min.js
 
 dist/base1/cockpit.min.css.map: dist/base1/cockpit.min.css
 
-dist/base1/patternfly.min.css.map: dist/base1/patternfly.min.css
-
-# Modify the patternfly files appropriately
-#
-# PatternFly 4 base.css: remove external font declarations, re-using the ones from our PatternFly 3 patternfly.min.css above for now
-# once that goes away, use the above seddery instead
-dist/base1/react-core-base.css:
-	$(AM_V_GEN) $(MKDIR_P) $(dir $@) && \
-	$(AM_V_GEN) sed '/^@font-face/,/^$$/d' $(srcdir)/node_modules/@patternfly/react-core/dist/styles/base.css > $@.tmp && mv $@.tmp $@
-
-# PatternFly 3 with hacked CSS to look like PF4
-dist/base1/patternfly.css.map: dist/base1/patternfly-base.css
-dist/base1/patternfly-base.css: src/base1/patternfly-overrides.less dist/base1/react-core-base.css
+# PatternFly 3 with hacked CSS to look like PF4 plus the PF4 base
+dist/base1/patternfly.css.map: dist/base1/patternfly.css
+dist/base1/patternfly.css: src/base1/patternfly-cockpit.scss
 	$(AM_V_GEN) $(MKDIR_P) $(dir $@) && \
-	F=$(srcdir)/node_modules/patternfly/dist/less/patternfly.less && \
-	sed '/import "variables.less"/d' $$F > $$F.patched && \
-	F_EXTRA=$(srcdir)/node_modules/patternfly/dist/less/patternfly-additions.less && \
-	cat $$F.patched $$F_EXTRA > $$F.tmp && \
-	echo -e '@import "../../../../src/base1/patternfly-overrides.less";\n@import "../../../../src/base1/patternfly-overrides-variables.less";' >> $$F.tmp && \
-	$(LESSC) --source-map=dist/base1/patternfly.css.map --source-map-include-source $$F.tmp $@.tmp && \
-	sed -i -f $(srcdir)/tools/patternfly.sed $@.tmp && \
-	rm $$F.tmp $$F.patched && \
-	$(MV) $@.tmp $@
+        $(SASS) $< $@ --source-map \
+			--source-map-urls absolute \
+			--style compressed \
+			--load-path=$(srcdir)/node_modules \
+			--load-path=$(srcdir)/node_modules/patternfly/dist/sass \
+			--load-path=$(srcdir)/node_modules/bootstrap-sass/assets/stylesheets \
+			--load-path=$(srcdir)/node_modules/font-awesome-sass/assets/stylesheets && \
+	sed -i -f $(srcdir)/tools/patternfly.sed $@
```

Comment:
Why do we still need the seddery here? The webpack config now uses the string-replace loader, and that looks by and large equivalent to what the sed is doing. Either you restrict that commit to only less → sass conversion, and do sed → string-replace-loader in a separate commit, or you keep it in one, but then the whole seddery should disappear. Or am I overlooking something?

Reply:
we still need to generate dist/base1/patternfly.css for a while until all project using it are ported to the new approach.
Since webpack does not generate dist/base1/patternfly.css (there is not entry point for it) we need to generate it here in the makefile and thus keep the sed.

Reply:
IOW I understood that we will keep shipping patternfly.css in cockpit-ws for a while.
For this purpose we need to maintain the Makefile logic. (either create another webpack entrypoint)

Reply:
Ah, of course -- thanks! I missed that webpack wasn't building that.